model_trainer.yaml

```diff
@@ -46,23 +51,23 @@ learning_objectives:
 
 
 training:
-  # Fraction of the data to use for training. Set to 1 to skip evaluation:
-  train_fraction: 0.8
+  # Fraction of the data to use for training. Set to 1 to skip evaluation or plp to use split from PLP in person sequence data:
+  train_fraction: plp
```

Comment:
Maybe it's unnecessary to mention this option in the docs. It's only the PLP that can use it.